public/wp-content/themes/newsup-child/functions.php

```diff
@@ -128,4 +128,90 @@ function newsup_banner_exclusive_posts()  {
     }
 endif;
 
+// This function is called on the home page for the featured
+// posts to be displayed on an Owl Carousel content section.
+// Similarly to the above function, we expect to simply
+// override the WP_Query object.
+if (!function_exists('newsup_front_page_banner_section')) :
+    /**
+     *
+     * @since Newsup
+     *
+     */
+    function newsup_front_page_banner_section()
+    {
+        if (is_front_page() || is_home()) {
+        $newsup_enable_main_slider = newsup_get_option('show_main_news_section');
+        $select_vertical_slider_news_category = newsup_get_option('select_vertical_slider_news_category');
+        $vertical_slider_number_of_slides = newsup_get_option('vertical_slider_number_of_slides');
+        $all_posts_vertical = newsup_get_posts($vertical_slider_number_of_slides, $select_vertical_slider_news_category);
+        if ($newsup_enable_main_slider):
+
+            $main_banner_section_background_image = newsup_get_option('main_banner_section_background_image');
+            $main_banner_section_background_image_url = wp_get_attachment_image_src($main_banner_section_background_image, 'full');
+        if(!empty($main_banner_section_background_image)){ ?>
+             <section class="mg-fea-area over" style="background-image:url('<?php echo $main_banner_section_background_image_url[0]; ?>');">
+        <?php }else{ ?>
+            <section class="mg-fea-area">
+        <?php  } ?>
+            <div class="overlay">
+                <div class="container-fluid">
+                    <div class="">
+                        <div class="col-md-8">
+                            <div class="row">
+                                <div id="homemain"class="homemain owl-carousel mr-bot60 pd-r-10">
+                                  <?php
+                                  // This code was pulled from ../newsup/inc/hooks/blocks/block-banner-list.php
+                                  // and controls "The Loop" which populates the Home Feature Post Banner.
+                                  $newsup_slider_category = newsup_get_option('select_slider_news_category');
+                                  $newsup_number_of_slides = newsup_get_option('number_of_slides');
+                                  $newsup_all_posts_main = newsup_get_posts($newsup_number_of_slides, $newsup_slider_category);
```

Comment:
This is the beginning of the template. This template was taken from another file and pasted here to simplify the process of making this change.